Implement Opus decoder

<!--
  Come join our community on Slack https://pion.ly/slack for quicker support and more resources

  We also provide a Knowledge Base https://pion.ly/knowledge-base that covers Pion and general WebRTC
  
  We maintain a FAQ of common issues and questions https://pion.ly/knowledge-base/pion-basics/faq/

  Please answer these questions before submitting your issue. Thanks!
-->

## Summary

When I get audio tracks on the OnTrack function I'm decoding them with 	"gopkg.in/hraban/opus.v2" opus module like so:
```
func (serverPeer *ServerPeer) onTrack(track *webrtc.TrackRemote, r *webrtc.RTPReceiver) {
	sampleRate := 48000
	channels := 2

	switch track.Kind() {
	case webrtc.RTPCodecTypeAudio:
		{

			dec, err := opus.NewDecoder(sampleRate, channels)
			if err != nil {
				panic(err)
			}

			for {
				data, _, err := track.ReadRTP()
				if err == io.EOF {
					break
				}
				if err != nil {
					panic(err)
				}
				if len(data.Payload) == 0 {
					continue
				}

				fps := 60
				frameSize := channels * sampleRate * fps / 1000 //5760
				pcm := make([]float32, frameSize)
				n, err := dec.DecodeFloat32(data.Payload, pcm)
			}
		}
	default:
		fmt.Println("Unsupported track.kind", track.Kind().String())
	}
```

I also use this module's opus package to add codecs to my MediaEngine. Therefore, I get the following exception:

```
/usr/local/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: /tmp/go-link-3022619521/000023.o: in function `bridge_encoder_set_bitrate':
/home/user/go/pkg/mod/github.com/pion/mediadevices@v0.2.0/pkg/codec/opus/opus.go:18: multiple definition of `bridge_encoder_set_bitrate'; /tmp/go-link-3022619521/000002.o:/home/user/go/pkg/mod/gopkg.in/hraban/opus.v2@v2.0.0-20210415224706-ab1467d63813/encoder.go:37: first defined here
collect2: error: ld returned 1 exit status
```

## Motivation

Im trying to decode the RTP packet which is in Opus format, Then I want to send it with Pulseaudio to another device.
Pulseaudio doesn't support sending Opus formatted packets. So I have to decode it. 

## Describe alternatives you've considered

Commenting out the `bridge_encoder_set_bitrate` function in one of the files to make it define only once. It worked of course, but I really don't want to do it.

--- a/pkg/codec/opus/opus.go
+++ b/pkg/codec/opus/opus.go
@@ -110,7 +110,7 @@ func (e *encoder) Read() ([]byte, func(), error) {
 }
 
 func (e *encoder) SetBitRate(bitRate int) error {
-	cerror := C.set_encoder_bitrate(
+	cerror := C.pion_set_encoder_bitrate(
 		e.engine,
 		C.int(bitRate),
 	)

--- a/pkg/codec/opus/opus.go
+++ b/pkg/codec/opus/opus.go
@@ -14,7 +14,7 @@ import (
 /*
 #include <opus.h>
 
-int bridge_encoder_set_bitrate(OpusEncoder *e, opus_int32 bitrate)
+int set_encoder_bitrate(OpusEncoder *e, opus_int32 bitrate)
 {
 	return opus_encoder_ctl(e, OPUS_SET_BITRATE(bitrate));
 }
@@ -110,7 +110,7 @@ func (e *encoder) Read() ([]byte, func(), error) {
 }
 
 func (e *encoder) SetBitRate(bitRate int) error {
-	cerror := C.bridge_encoder_set_bitrate(
+	cerror := C.set_encoder_bitrate(
 		e.engine,
 		C.int(bitRate),
 	)

--- a/pkg/codec/opus/opus.go
+++ b/pkg/codec/opus/opus.go
@@ -14,7 +14,7 @@ import (
 /*
 #include <opus.h>
 
-int set_encoder_bitrate(OpusEncoder *e, opus_int32 bitrate)
+int pion_set_encoder_bitrate(OpusEncoder *e, opus_int32 bitrate)
 {
 	return opus_encoder_ctl(e, OPUS_SET_BITRATE(bitrate));
 }

